### PR TITLE
Mobile version setup contact form

### PR DESCRIPTION
**Pull request by sharon-odhiambo**
Add contact-form website dynamics
Add form validations
Style contact form using css
Fix stylelint errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,232 +1,395 @@
...
+                  /></span>
+                </p>
+              </div>
+            </li>
+          </ul>
+        </div>
+      </section>
+    </section>
+    <section class="form">
+      <form action="/my-form-page" method="post">
```
Comment (zieeco) [inline index.html:234]: **[REQUIRED CHANGES]**

- Please, kindly ensure you've used the right [formspree](https://formspree.io/html) to enable the submission of a form on your portfolio website, right now, your form is not submitting as expected.

File: index.html [hunk 2]
```diff
@@ -1,232 +1,395 @@
...
+            <li>
+              <label for="mail"></label>
+              <input type="email" id="mail" name="user_mail" placeholder="your email " required/>
+            </li>
+            <li>
+              <label for="msg"></label>
+              <textarea class="text" id="msg" name="user_message" maxlength="500" placeholder="write something here" required></textarea>
+            </li>
+            <li>
+              <button class="but" type="submit">Get in Touch</button>
```
Comment (zieeco) [inline index.html]: **[REQUIRED CHANGED]**


- Kindly add the interactions on **ALL** your buttons to indicate the hover, active, enabled, and disabled state to improve the  user experience and to meet the project requirement below:


buton

File: index.html [hunk 3]
```diff
@@ -1,232 +1,395 @@
...
+                target="_blank"
+                rel="noopener"
+                class="social-media-link"
+                ><img src="./images/angelist.svg" alt="Github" />
+              </a>
+            </li>
+          </ul>
+          <div>
+            <button type="button" class="button">
+              <span>Get my Resume</span>
```
Comment (zieeco) [inline index.html]: **[REQUIRED CHANGES]**

- Please, kindly improve your styling on all the `get my resume` buttons to match the Figma design below:



**figma**
gmr





**your button**
bone

File: index.html [hunk 4]
```diff
@@ -1,232 +1,395 @@
...
+            </div>
+            <div>
+              <ul class="languages-works-three">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            <button type="button" class="button">
+              <span>See Project</span>
```
Comment (zieeco) [inline index.html]: **[REQUIRED CHANGES]**

- Please, kindly improve your styling on **ALL** the `see project` buttons to match the Figma design below:


**figma**
sp


**your button**
btwo

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @sharon-odhiambo  and @Hirwacardinalloic 


Great work with the contact form 👏 
You've done well implementing it, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @sharon-odhiambo ✋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 🚨 


 ## Highlights ⚡ 

- [x] Good job implementing the previous reviewer's requirements.

 ## Required Changes ♻️

- [ ] Your contact section doesn't quite look like the template design.

Screen Shot 2022-06-09 at 12 37 07

Please fix it.

_Check the comments under the review._

## Optional suggestions 📂 

- [ ] [OPTIONAL] Your design doesn't quite look like the template design.

Screen Shot 2022-06-09 at 12 31 07

Screen Shot 2022-06-09 at 12 35 35

Screen Shot 2022-06-09 at 12 36 30

Please check the Figma template and make sure you implement all the necessary changes to make it look as similar as possible. Some things to think about: margins, padding, fonts, colors, and button interaction styling, among other things.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @sharon-odhiambo  🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**